File: java/Soal-2/Makhluk.java
public class Makhluk {
    private String jenis;

    Makhluk() {
        this.jenis="";
    }

    Makhluk(String jns) {
        this.jenis=jns;
    }

    public void setJenis(String jenis) {
        this.jenis=jenis;
    }

    public String getJenis() {
        return this.jenis;
    }

    public void printInfo() {
        System.out.println("Jenis = " + this.jenis);
    }
}

File: py/Soal-2/makhluk.py
class Makhluk:
    def __init__(self) :
        self.__jenis=""

    def __init__(self, jns) :
        self.__jenis=jns

    def setJenis(self,jenis) :
        self.__jenis=jenis

    def getJenis(self) :
        return self.__jenis

    def printInfo(self) :
        print("Jenis\t\t:" + self.__jenis)
